CIP-XXXX/README.md

```diff
@@ -0,0 +1,65 @@
+---
+CIP: XXXX
+Title: Standalone transaction serialization without the `isValid` flag
+
```

Comment:
```suggestion
Title: Remove the `isValid` flag from standalone transaction serialization
```
Of course @teodanciu @lehins if you can make it even shorter (without it being ambiguous) it would even better: since CIP titles tend to be short & usually shorter than this.

@Ryun1 @Crypto2099 @perturbing I don't think we have to remove the literal `` ` `` delimiters from CIP titles anymore.  This was mainly to prevent bugs in Docusaurus when the content was copied onto the Developer Portal... now not a problem since it links to `cips.cardano.org` which interprets the backticks correctly (I think).  If indeed it's not a problem after this is merged & rendered there, I'll remove the backtick prohibition from CIP-0001 (and CIP-9999 if it's also there) at the next all-around update.

Reply:
Mainly resolved with https://github.com/cardano-foundation/CIPs/pull/1089/commits/202edf555133a797383f6747c539b464bd907ad8 (@teodanciu you are also welcome to commit the changes from here in the UI so the resolution happens automatically... FYI there is still also the extra blank line in the header that would also have been corrected if the suggested change had been committed this way.)